src/apps/bristleback_apps/aanderaa/user_code/user_code.cpp

```diff
@@ -21,13 +21,51 @@
 
 #define LED_ON_TIME_MS 20
 #define LED_PERIOD_MS 1000
-#define AANDERAA_WATCHDOG_TIMEOUT_MS (5 * 1000)
+
 #define AANDERAA_WATCHDOG_MAX_TRIGGERS (3)
 #define AANDERAA_WATCHDOG_ID "Aanderaa"
 #define AANDERAA_RAW_LOG "aanderaa_raw.log"
 #define AANDERAA_RESET_TIME_MS                                                                 \
   (500) // https://www.aanderaa.com/media/pdfs/td266-zpulse-dcs-4420-4830-4520-4930.pdf P.90
 
+/// Default mote configurations and local variables
+// How many minutes to collect readings for before shipping an aggregation.
+#define DEFAULT_CURRENT_AGG_PERIOD_MIN (10.0)
+static float current_agg_period_min = DEFAULT_CURRENT_AGG_PERIOD_MIN;
+// inline helper to turn this into milliseconds
+#define CURRENT_AGG_PERIOD_MS ((double)current_agg_period_min * 60 * 1000)
+
+// How many samples to leave out of aggregation during sensor initialization.
+// - If using reading intervals <5 seconds, recommend 2, otherwise 0.
+// It takes time for some internal Aanderaa init, and we're not parsing the "*" characters to infer errors yet.
+#define DEFAULT_N_SKIP_READINGS (0)
+static uint32_t n_skip_readings = DEFAULT_N_SKIP_READINGS;
+static uint8_t readings_skipped = 0;
+
+// The reading interval in ms the Aanderaa is configured for.
+// - Currently, must take care to manually set Aanderaa and this setting to the same values.
+#define DEFAULT_CURRENT_READING_INTERVAL_MS (60*1000)
+static uint32_t reading_interval_ms = DEFAULT_CURRENT_READING_INTERVAL_MS;
```

Comment:
The combination of "must take care" and 60s here vs. 2s and 1s mentioned in PR description feels like a yellow flag. What have we currently decided is actually the best reading interval? Can we make it the default? Can we set the interval in the Aanderaa sensor from firmware, or does it require a different tool?

Reply:
Sorry, PR comments were auto-generated from commit tags - the 1s and 2s are no longer relevant. 60s is the decided default for DVT2.
In principal we can eventually set the reading interval on the Aanderaa in firmware, but this will require a more comprehensive driver to be developed.